lib/validic/response.rb

```diff
@@ -11,5 +13,23 @@ def initialize(summary, response)
         @attributes = response
       end
     end
+
+    def next
+      extend Validic::REST::Utils
```

Comment:
i need the methods in Validic::REST::Utils in the next and previous methods but because of dependency issues. This is the only way i can come up at the moment.